core/arch/arm/plat-ti/conf.mk

```diff
@@ -15,6 +15,7 @@ $(call force,CFG_PL310_LOCKED,y)
 $(call force,CFG_SECURE_TIME_SOURCE_REE,y)
 arm32-platform-cpuarch		:= cortex-a9
 else
+$(call force,CFG_OTP_SUPPORT,y)
```

Comment:
Nitpicking: should this be `CFG_OTP_SUPPORT ?= y` instead? (so that the option may be disabled at build time: `make ... CFG_OTP_SUPPORT=n`). Now, if you think it doesn't make much sense, I'm perfectly OK with `force`.